cmd/conf/chain.go

```diff
@@ -52,27 +57,44 @@ func (c *ParentChainConfig) Validate() error {
 	return c.Connection.Validate()
 }
 
+// lint:require-exhaustive-initialization
 type L2Config struct {
-	ID        uint64                   `koanf:"id"`
-	Name      string                   `koanf:"name"`
-	InfoFiles []string                 `koanf:"info-files"`
-	InfoJson  string                   `koanf:"info-json"`
-	DevWallet genericconf.WalletConfig `koanf:"dev-wallet"`
+	ID               uint64                   `koanf:"id"`
+	Name             string                   `koanf:"name"`
+	InfoFiles        []string                 `koanf:"info-files"`
+	InfoJson         string                   `koanf:"info-json"`
+	InitialL1BaseFee string                   `koanf:"initial-l1base-fee"`
+	DevWallet        genericconf.WalletConfig `koanf:"dev-wallet"`
+}
+
+func (c *L2Config) InitialL1BaseFeeParsed() (*big.Int, error) {
+	if c.InitialL1BaseFee == "" {
+		return params.DefaultInitialL1BaseFee, nil
+	}
+
+	parsed, success := big.NewInt(0).SetString(c.InitialL1BaseFee, 10)
+	if !success {
+		log.Error("Failed to parse L1 BaseFee for L2 config (passed with `initial-l1base-fee`)", "config", c.InitialL1BaseFee)
```

Comment:
nitpick
```suggestion
		log.Error("Failed to parse L1 BaseFee for L2 config (passed with `initial-l1base-fee` flag)", "config", c.InitialL1BaseFee)
```

Reply:
added "flag" to the error message in [01e9108](https://github.com/OffchainLabs/nitro/pull/4171/commits/01e91084db94789b8779e184c9eaaa00af1e0bff)